modules/notes.php

```diff
@@ -104,6 +104,8 @@ function action_init() {
 
 	function styles_and_scripts() {
 		$is_rtl = is_rtl();
+		$style_handles  = [];
```

Comment:
While we can use short array syntax in PHP 5.6+, WordPress core opted against recommending their use in their standards. I would consequently recommend that we follow Core's lead here.

You can find out more here:
https://make.wordpress.org/core/2019/07/12/php-coding-standards-changes/

Reply:
Sure I'll change it to `array()`. I was aware of that decision for core, but we've come to a different conclusion for the AMP plugin: 

https://github.com/ampproject/amp-wp/pull/2767
https://github.com/ampproject/amp-wp/pull/3056#issuecomment-523610524